articles/active-directory/saas-apps/github-provisioning-tutorial.md

```diff
@@ -20,7 +20,7 @@ ms.collection: M365-identity-device-management
 ---
 # Tutorial: Configure GitHub for automatic user provisioning
 
-The objective of this tutorial is to show you the steps you need to perform in GitHub and Azure AD to automatically provision and de-provision user accounts from Azure AD to GitHub.
+The objective of this tutorial is to show you the steps required to automatically provision and de-provision organization membership of user accounts from Azure AD to GitHub.
```

Comment:
not sure if organization membership of is needed / grammatically correct. 

changing you need to required is fine. removing the perform section is fine.